cl8/utils/tests/test_vcard_util.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from vobject.vcard import Name
+
+from .. import vcard_util
+
+
+class TestVcardUtil:
+    def test_full_name_to_name(self):
+        all_in_given_name = [
+            "",
+            "Biscuit",
+            "Dr.",
+            "Dr. Gravy Biscuit",
+            "Sweet Gravy Biscuit",
+            "Ms. Dr. Cheese Gravy Biscuit Jr. M.D.",
+            "Biscuit Jr.",
+            "宮本 茂",
+            "أبو بکر محمد بن زکریاء الرازي",
+        ]
+        for full_name in all_in_given_name:
+            actual = vcard_util.full_name_to_name(full_name)
+            expected = Name(given=full_name)
+            assert actual == expected
+
+        split = [
+            ("Gravy", "Biscuit"),
+            ("Großer", "Keks"),
+        ]
+        for given, family in split:
+            full_name = f"{given} {family}"
+            actual = vcard_util.full_name_to_name(full_name)
+            expected = Name(given=given, family=family)
+            assert actual == expected
+
+    def test_content_disposition_filename(self):
```

Comment:
Same here. I know the `content_disposition_filename()` is quite well documented, but a one liner along the lines of:

> check that we can create a valid filename for a vcard, based on a profile

Or something to that effect.

Reply:
Done in 398261fabd55a5cb670fb9c88dd1ad6a43e20a06.